IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_CommunicationsApplianceTypeOpticalLineTerminal/DocPropertySet.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_CommunicationsApplianceTypeOpticalLineTerminal" UniqueId="5d11e497-ab53-a92b-61a8-8a54a11535b3" ApplicableType="IfcCommunicationsAppliance/USERDEFINED" PropertySetType="PSET_TYPEDRIVENOVERRIDE">
+	<Properties>
+		<DocProperty xsi:nil="true" href="NumberOfSlots_3ol50yqfz8ZAv5ApU0vpo5" />
+		<DocProperty xsi:nil="true" href="NumberOfInterfaces_0x8qDQiJjFd9ZlduDoHwXH" />
+		<DocProperty xsi:nil="true" href="NumberOfOccupiedSlots_0BB9fPQzj1QRSm5RHDh4up" />
```

Comment:
Is NumberOfOccupiedSlots an instance property?  Although an instance propertyset for one property isn't ideal.

Reply:
It is actually an instance property. I keep it here because I don't want to create one Pset just for this property